src/components/PropDrawer/index.jsx

```diff
@@ -41,13 +37,7 @@ export default function TemporaryDrawer(props) {
           open={state["right"]}
           onClose={toggleDrawer("right", false)}
         >
-          <div
-            style={{
-              width: "70vw",
-              maxWidth: 700,
-              paddingLeft: "1rem",
-            }}
-          >
+          <div className={styles.propdrawer__child}>
```

Comment:
just a suggestion we can use camel casing here 😅 ... I remembered seeing a pr in which the maintainer asked to use camel casing.